changelog/front-commerce-3.8-2.33.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: "Release: 3.8 & 2.33"
+authors: pierre
+date: 2024-10-10T10:10
+hide_table_of_contents: true
+---
+
+## Gezy major features and customer-driven improvements
+
+As more projects go live with Front-Commerce 3.x or upgrade to the latest
+versions, we've received valuable feedback from our customers. **Front-Commerce
+3.8 incorporates core improvements and features to support projects with diverse
+and specific requirements.**
+
+This release marks a major milestone for our Gezy integration, introducing core
+e-commerce functionalities. We're excited to announce that **you can now
+confidently start a project with Front-Commerce and Gezy**, as we've implemented
+essential features and are now focusing on advanced capabilities!
+
+As always, this release includes bug fixes that enhance the overall stability,
+DX and performance of the platform.
+
+<!-- truncate -->
+
+## Gezy: key features implemented
+
+Building on the foundations laid in version 3.7, we've successfully implemented
+core e-commerce functionalities for the Gezy platform. **This release provides
+all the essential features for guide a visitor from browsing your catalog to
+becoming a customer with their first completed order!**
```

Comment:
This part does not make sence:  `features for guide a visitor from browsing your catalog`